tensorflow_lite_support/cc/task/processor/image_postprocessor.cc

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow_lite_support/cc/task/processor/image_postprocessor.h"
+
+namespace tflite {
+namespace task {
+namespace processor {
+
+namespace {
+
+using ::absl::StatusCode;
+using ::tflite::metadata::ModelMetadataExtractor;
+using ::tflite::support::CreateStatusWithPayload;
+using ::tflite::support::StatusOr;
+using ::tflite::support::TfLiteSupportStatus;
+
+}  // namespace
+
+/* static */
+tflite::support::StatusOr<std::unique_ptr<ImagePostprocessor>>
+ImagePostprocessor::Create(core::TfLiteEngine* engine, const int output_index,
+                           const int input_index) {
+  ASSIGN_OR_RETURN(auto processor,
+                   Processor::Create<ImagePostprocessor>(
+                       /* num_expected_tensors = */ 1, engine, {output_index},
+                       /* requires_metadata = */ false));
+
+  RETURN_IF_ERROR(processor->Init(input_index));
+  return processor;
+}
+
+absl::Status ImagePostprocessor::Init(const int input_index) {
+  if (input_index == -1) {
+    return tflite::support::CreateStatusWithPayload(
+        absl::StatusCode::kInvalidArgument,
+        absl::StrFormat("Input image tensor not set. Input index found: %d",
+                        input_index),
+        tflite::support::TfLiteSupportStatus::kInputTensorNotFoundError);
+  }
+  ASSIGN_OR_RETURN(
+      auto output_specs,
+      vision::BuildImageTensorSpecs(*engine_->interpreter(),
+                                    *engine_->metadata_extractor(), false));
+  options_ = std::make_unique<vision::NormalizationOptions>(
+      output_specs.normalization_options.value());
+  return absl::OkStatus();
+}
+
+absl::StatusOr<vision::FrameBuffer> ImagePostprocessor::Postprocess() {
+  has_uint8_output_ = GetTensor()->type == kTfLiteUInt8;
+  const int kRgbPixelBytes = 3;
```

Comment:
Create a constexpr, such as `constexpr int kRgbPixelBytes = 3;', and put it in the anonymous namespace with other `using` statements.